estate/models/estate_property_offer.py

```diff
@@ -0,0 +1,58 @@
+from odoo import models, fields, api, _
+from datetime import timedelta
+from odoo.exceptions import UserError
+
+
+class EstatePropertyOffer(models.Model):
+    _name = "estate.property.offer"
+    _description = "Property offer"
+    _order = "price desc"
+
+    price = fields.Float(required=True)
+    status = fields.Selection(
+        selection=[
+            ("accepted", "Accepted"),
+            ("refused", "Refused")
+            ]
+        )
```

Comment:
We always need to run the pre-commit-vauxoo linter at the end to avoid getting indentation or linter problems

Or check your custom IDE linter, maybe its not set to check Python files?

```suggestion
    status = fields.Selection(
        selection=[
            ("accepted", "Accepted"),
            ("refused", "Refused")
        ]
    )
```